docs/source/conf.py

```diff
@@ -44,9 +44,10 @@
 # ones.
 extensions = [
     'sphinx.ext.autodoc',
+    "sphinx.ext.autosummary",
```

Comment:
Inconsistent use of double quotes. See https://github.com/pre-commit/pre-commit-hooks/blob/main/README.md#double-quote-string-fixer